src/main/java/net/sf/mzmine/modules/rawdatamethods/peakpicking/massdetection/exactmass/ExactMassDetector.java

```diff
@@ -37,50 +38,57 @@
     /**
      * @see net.sf.mzmine.modules.peakpicking.threestep.massdetection.MassDetector#getMassValues(net.sf.mzmine.datamodel.Scan)
      */
-    public DataPoint[] getMassValues(Scan scan, ParameterSet parameters) {
+	public String getDescription(String job, String str) { return str;}
+	
+	public String filterTargetName(String name) { return name; }
+	public String startMassValuesJob(RawDataFile raw, String targetName, ParameterSet parameters, int scanCount) { return null; }
+
+	public DataPoint[] getMassValues(Scan scan, boolean selected, String job, ParameterSet parameters) {
+    	if (!selected)	// only process selected scans
+    		return null;
+	
+		double noiseLevel = parameters.getParameter(
```

Comment:
Fix the indentation so that the bulk of this function doesn't appear to fall under the ```if(!selected)``` statement.

You should probably also make sure you load and use the proper formatting style, which is available in the ```mzmine-eclipse-code-formatter.xml``` file. Perhaps do a single commit that is only formatting updates.